contentcuration/contentcuration/frontend/shared/vuex/file/__tests__/module.spec.js

```diff
@@ -122,5 +145,68 @@ describe('file store', () => {
         });
       });
     });
+    describe('H5P content file extract metadata', () => {
+      it('getH5PMetadata should check for h5p.json file', () => {
+        const zip = new JSZip();
+        const metadata = {};
+        return zip.generateAsync({ type: 'blob' }).then(async function(h5pBlob) {
+          await expect(Promise.resolve(getH5PMetadata(h5pBlob, metadata))).resolves.toStrictEqual(
+            new Error('h5p.json not found in the H5P file.')
```

Comment:
```suggestion
          await expect(getH5PMetadata(h5pBlob, metadata)).rejects.toThrow(
            new Error('h5p.json not found in the H5P file.')
```

I think according to the docs this might be more how we should be checking for this? https://jestjs.io/docs/expect#rejects